### DISCUSSION TITLE
Exclude Plugins from Cloud Sync

### DISCUSSION THREAD

**Discussion by abanksss** [runelite/runelite#18059] [Ideas]
I play osrs primarily on two different devices, my pc and my laptop. My pc is much more powerful so I like to use stuff like 117 HD and other plugins that don't run well on my laptop. In the past I have used two separate profiles for my laptop and pc but that prevents everything else that I do want cloud synced from transferring over. It would be nice if I could exclude 117 and the GPU plugins from cloud syncing. This could be done by right clicking the plugin name in the list of installed plugins and clicking a button to exclude them.

**Comment by ayanami3** [comment]
Seconded, I have a similar setup with laptop and desktop. Wanted to sync my bank tags only to realize my gpu settings were also synced

**Reply by abanksss** [reply]
I did actually find a nice plugin recently that helps with my setup. Its called plugin presets and it lets me save my plugin settings for each device and toggle them easily. There is even the option of whether you want a preset to sync to your runelite account or just stay local on that device. I don't think there is an option to set a default preset per device but I was considering putting in a feature request.
